railgun/config/base/kong-ingress-2x.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: ingress-kong
+  namespace: kong
+spec:
+  template:
+    spec:
+      containers:
+      - name: ingress-controller
+        image: kong/kubernetes-ingress-controller:2.0.0-alpha.2
```

Comment:
should not this be kic beta release ?

Reply:
Not until that image actually exists, no. Updating these with the upcoming release version is part of our release process; when we're preparing to release 2.0.0-beta.1 we'll change it.

Reply:
NOTE: if we have several places where we need to manually update versions we should note that in the RELEASE.md notes, and create follow up Github issues to later automate that.